twitter.js

```diff
@@ -84,11 +85,18 @@ class Twitter {
    */
   static _handleResponse(response) {
     const headers = response.headers.raw(); // TODO: see #44
+
     // Return empty response on 204 "No content"
     if (response.status === 204)
       return {
         _headers: headers,
       };
+
+    // Endpoint has no response, don't try to parse it
+    if (response.headers.get('content-length') === '0') return {
+      _headers: headers,
+    };
```

Comment:
After resolving merge conflicts, can you double-check to see if you still need this check? A recently-merged PR also touches `_handleReponse`.

I'm not in love with this check for no response, since it seems a bit flaky (compared to the `response.status` check above), but I'm no web expert so this might be our best option.

Reply:
Conflicts resolved!

You might have missed my explanation in the pull request description, but the reason `media/metadata/create` crashed was due to it responding with `HTTP 200` but no content. Therefore it crashed when it tried to decode the empty response as JSON